library/Zend/Db/Sql/Select.php

```diff
@@ -44,6 +44,10 @@ class Select extends AbstractSql implements SqlInterface, PreparableSqlInterface
     const JOIN_OUTER = 'outer';
     const JOIN_LEFT = 'left';
     const JOIN_RIGHT = 'right';
+    const JOIN_RIGHT_OUTER = 'outer right';
+    const JOIN_LEFT_OUTER  = 'outer left';
+    const JOIN_RIGHT_INNER = 'inner right';
+    const JOIN_LEFT_INNER = 'inner left';
```

Comment:
Not to be pedantic, but... considering the actual word order, shouldn't these be:

- `JOIN_OUTER_RIGHT`
- `JOIN_INNER_LEFT`
- etc.

? Right now, the constant names do not mirror the actual usage, which could be confusing, particularly if you were to use the constant value elsewhere.